app/controllers/movies_controller.rb

```diff
@@ -13,8 +15,12 @@ def index
   end
 
   def show 
-    @movie = Movie.includes(:videos).where(id: params[:id])
+    @movie = Movie.includes(:videos).where(id: params[:id]).first
 
-    render json: @movie, :include => [:videos], status: :ok
+    if @movie
+      render json: @movie, :include => [:videos], status: :ok
+    else
+      render json: { status: 'ERROR', message: 'The movie does not exist' }, status: :not_found
```

Comment:
Why do you need 2 statuses here? Error and :not_found. Checking against the status code should be enough.